server/conf/evolutions/default/47.sql

```diff
@@ -0,0 +1,8 @@
+# --- !Ups
+
+alter table programs add column program_type varchar;
+update programs set program_type = 'DEFAULT' where program_type is null;
```

Comment:
**question (blocking):** `'DEFAULT'` is all-caps here but the enum `dbValue` is lowercase. Is that intentional?

Reply:
Oops nope, I just switched from all caps to lower case in the enum and forgot to make the switch here as well. thanks!